### DISCUSSION TITLE
Fluxcd auto image update

### DISCUSSION THREAD

**Discussion by ddaluka** [fluxcd/flux2#3957] [Q&A]
Hi All,

I have recently started using fluxcd and currently using it without its git integration i.e. flux was directly installed in GKE cluster using its install.yaml. 

I am deploying the helm charts using helmrelease which are sitting in the GCP artifact registry (OCI). As soon as I upload a new chart in registry, fluxcd detects that deploys it in my cluster. But now I am looking for auto image updates using fluxcd. 

We were using Keel so far. But want to move everything to Fluxcd. I have created image policy and image repository manifests but it seems like until we integrate fluxcd with Git, this auto image update can not be achieved.  

Currently, I am using keel and fluxcd together. Keel looks for the new version of an image in registry and updates it but fluxcd sees a configuration drift and tries to deploy the same release again. Image tag we are using for now is 'latest'

**Comment by makkes** [comment]
There's several things to keep in mind:

1. `latest` is not an appropriate tag to use in a policy as the tags need to be unique. See [this guide on how to come up with sortable image tags](https://fluxcd.io/flux/guides/sortable-image-tags/) for more details.
2. Yes, you can't use `ImageUpdateAutomation` without `GitRepository` if that's what you're referring to. You need to install at least the CRD in your cluster. source-controller does not need to be running, though.

**Reply by ddaluka** [reply]
Hi @makkes , 

Using fluxcd, we are trying to move to specific versions instead of latest and that's where looking for auto image update solution using image policy 

I have source controller CRD deployed but we are using helmrepository instead of GitRepository .
